src/containers/pages/InterventionPlan/PlanningView/helpers/OpenSRPPlansList/index.tsx

```diff
@@ -10,21 +10,28 @@ import {
   renderInFilterFactory,
 } from '../../../../../../components/Table/DrillDownFilters/utils';
 import { NoDataComponent } from '../../../../../../components/Table/NoDataComponent';
-import { PLAN_RECORD_BY_ID, QUERY_PARAM_TITLE } from '../../../../../../constants';
-import { loadOpenSRPPlans } from '../../../../../../helpers/dataLoading/plans';
+import {
+  PLAN_RECORD_BY_ID,
+  QUERY_PARAM_TITLE,
+  QUERY_PARAM_USER,
+} from '../../../../../../constants';
+import {
+  loadOpenSRPPlans,
+  loadPlansByUserFilter,
+} from '../../../../../../helpers/dataLoading/plans';
 import { getQueryParams } from '../../../../../../helpers/utils';
 import { OpenSRPService } from '../../../../../../services/opensrp';
 
+import { displayError } from '../../../../../../helpers/errors';
+import { makePlansByUserNamesSelector } from '../../../../../../store/ducks/opensrp/planIdsByUser';
```

Comment:
is the planIdsByUser reducer registered elsewhere

Reply:
Had registered in different pages but have moved it only on this page.